File: src/main/java/kr/syeyoung/dungeonsguide/whosonline/api/messages/server/S01IsOnlineAck.java
package kr.syeyoung.dungeonsguide.whosonline.api.messages.server;

import kr.syeyoung.dungeonsguide.whosonline.api.messages.AbstractMessage;
import lombok.Data;

@Data
public class S01IsOnlineAck implements AbstractMessage {
    public final boolean is_online;
    public final String uuid;
    public final String nonce;

}

File: src/main/java/kr/syeyoung/dungeonsguide/whosonline/api/messages/server/S01IsOnlineAck.kt
package kr.syeyoung.dungeonsguide.whosonline.api.messages.server

import kr.syeyoung.dungeonsguide.whosonline.api.messages.AbstractMessage
import lombok.Data

@Data
class S01IsOnlineAck : AbstractMessage {
    @JvmField
    val is_online: Boolean
    @JvmField
    val uuid: String
    @JvmField
    val nonce: String

    constructor(is_online: Boolean, uuid: String, nonce: String) {
        this.is_online = is_online
        this.uuid = uuid
        this.nonce = nonce
    }
}